nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -291,111 +501,131 @@ def permuted_ols(tested_vars, target_vars, confounding_vars=None,
 
     Returns
     -------
-    pvals : array-like, shape=(n_regressors, n_descriptors)
-      Negative log10 p-values associated with the significance test of the
-      n_regressors explanatory variates against the n_descriptors target
-      variates. Family-wise corrected p-values.
+    neg_log10_pvals : array-like, shape=(n_regressors, n_descriptors)
+        Negative log10 p-values associated with the significance test of the
+        ``n_regressors`` explanatory variates against the ``n_descriptors``
+        target variates. Family-wise corrected p-values.
 
     score_orig_data : numpy.ndarray, shape=(n_regressors, n_descriptors)
-      t-statistic associated with the significance test of the n_regressors
-      explanatory variates against the n_descriptors target variates.
-      The ranks of the scores into the h0 distribution correspond to the
-      p-values.
+        T-statistics associated with the significance test of the
+        ``n_regressors`` explanatory variates against the ``n_descriptors``
+        target variates.
+        The ranks of the scores into the h0 distribution correspond to the
+        p-values.
 
-    h0_fmax : array-like, shape=(n_perm, )
-      Distribution of the (max) t-statistic under the null hypothesis
-      (obtained from the permutations). Array is sorted.
+    h0_fmax : array-like, shape=(n_regressors, n_perm)
+        Distribution of the (max) t-statistic under the null hypothesis
+        (obtained from the permutations). Array is sorted.
 
     References
     ----------
     .. [1] Anderson, M. J. & Robinson, J. (2001). Permutation tests for
-       linear models. Australian & New Zealand Journal of Statistics, 43(1), 75-88.
+       linear models. Australian & New Zealand Journal of Statistics, 43(1),
+       75-88.
 
     .. [2] Winkler, A. M. et al. (2014). Permutation inference for the general
        linear model. Neuroimage.
 
-    .. [3] Freedman, D. & Lane, D. (1983). A nonstochastic interpretation of reported
-       significance levels. J. Bus. Econ. Stats., 1(4), 292-298
+    .. [3] Freedman, D. & Lane, D. (1983). A nonstochastic interpretation of
+       reported significance levels. J. Bus. Econ. Stats., 1(4), 292-298
 
     """
     # initialize the seed of the random generator
     rng = check_random_state(random_state)
 
     # check n_jobs (number of CPUs)
     if n_jobs == 0:  # invalid according to joblib's conventions
-        raise ValueError("'n_jobs == 0' is not a valid choice. "
-                         "Please provide a positive number of CPUs, or -1 "
-                         "for all CPUs, or a negative number (-i) for "
-                         "'all but (i-1)' CPUs (joblib conventions).")
+        raise ValueError(
+            "'n_jobs == 0' is not a valid choice. "
+            "Please provide a positive number of CPUs, or -1 for all CPUs, "
+            "or a negative number (-i) for 'all but (i-1)' CPUs "
+            "(joblib conventions)."
+        )
```

Comment:
This conflicts with the docstring, which says that `n_jobs == 0` will use all CPUs.

Reply:
I opened #3182 to address this.